apps/etterlatte-behandling/src/main/kotlin/Application.kt

```diff
@@ -137,37 +93,4 @@ private fun Route.attachContekst(ds: DataSource) {
         }
         Kontekst.remove()
     }
-}
-
-class App(private val beanFactory: BeanFactory) {
-    fun run() {
-        beanFactory.grunnlagsendringshendelseJob()
-        embeddedServer(
-            CIO,
-            applicationEngineEnvironment {
-                modules.add { sikkerhetsModul() }
-                modules.add { module(beanFactory) }
-                connector { port = 8080 }
-            }
-        ).apply { setReady() }.start(true)
-        beanFactory.behandlingHendelser().nyHendelse.close()
-    }
-}
-
-private fun ventPaaNettverk() {
```

Comment:
Vi kan fjerne denne nå altså, den ble jo brukt så man kunne vente på linkerd sidecaren var oppe ellers så tryner appen.

Reply:
Hva brukes den sidecaren til egentlig? Er dette noe som er spesifikt for behandling?

Reply:
Aner ikke, du kan gjerne prøve uten for min del.

Reply:
Ryktes om at det er kafka, prøv å fjerne denne og deploye. Evt også uten det i configen, må vi ha noe delay så behandler vi det i konfigen.